packages/clarity-js/src/data/extract.ts

```diff
@@ -86,10 +88,16 @@ export function compute(): void {
                 let selectorData = selectors[key];
                 for (let s in selectorData) {
                     let selectorKey = parseInt(s);
-                    let nodes = document.querySelectorAll(selectorData[selectorKey]) as NodeListOf<HTMLElement>;
+                    let selector = selectorData[selectorKey];
+                    if (selector.startsWith(Constant.At)){
+                        shouldMask = true;
+                        selector = selector.substring(1, selector.length);
+                    }
+                    let nodes = document.querySelectorAll(selector) as NodeListOf<HTMLElement>;
```

Comment:
nit: we can avoid resetting selector value by doing selector.slice(1) in the querySelectorAll query directly